facade/pom.xml

```diff
@@ -139,6 +139,11 @@
             <artifactId>hibernate-core</artifactId>
             <scope>provided</scope>
         </dependency>
+        <dependency>
+            <groupId>org.jboss.resteasy</groupId>
+            <artifactId>resteasy-jaxrs</artifactId>
+            <scope>provided</scope>
+        </dependency>
```

Comment:
Why is this needed?

Reply:
ClientErrorException

Reply:
Maybe throw different exception. Wrap it in the rest-new module or write exception mapper?

Reply:
ClientErrorException should be in some javax.ws module (rs-api?)

Reply:
any new dependency is completely removed in most recent version of PR

Reply:
@janinko WDYT? I am fine, with the changes.